src/server/scripts/Kalimdor/BlackfathomDeeps/blackfathom_deeps.cpp

```diff
@@ -238,10 +239,41 @@ class npc_morridune : public CreatureScript
     }
 };
 
+class spell_subjugator_korul_darkness_calls : public SpellScriptLoader  // 151159
+{
+    public:
+        spell_subjugator_korul_darkness_calls() : SpellScriptLoader("spell_subjugator_korul_darkness_calls") { }
+
+        class spell_subjugator_korul_darkness_calls_SpellScript : public SpellScript
+        {
+            PrepareSpellScript(spell_subjugator_korul_darkness_calls_SpellScript);
+
+            void HandleScript(SpellEffIndex /*effIndex*/)
+            {
+                if (Creature* caster = GetCaster()->ToCreature())
```

Comment:
If you need the caster to be a creature (hint: you don't in this simple script) you should enforce it with a typeid check in Load(), not here